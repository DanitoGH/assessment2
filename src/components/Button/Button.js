import React from 'react';
import { Button } from 'react-bootstrap';


const button = ({ variant, value, onClick }) => (
     <Button variant={ variant }  onClick={() => onClick() } block>
         { value }
     </Button>
 );

 export default button;