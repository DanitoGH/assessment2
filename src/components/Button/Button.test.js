import { shallow }  from "enzyme";
import toJSON from "enzyme-to-json";

import Button from "./Button";


test("matches snapshot", () => {
    const wrapper = shallow(<Button />);

    expect(toJSON(wrapper)).toMatchSnapshot();
})

