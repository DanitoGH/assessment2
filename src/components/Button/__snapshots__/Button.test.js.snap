// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Button
  active={false}
  block={true}
  disabled={false}
  onClick={[Function]}
  variant="primary"
/>
`;
